File: Selenium/Project/Activity3.py
# Import required packages from selenium
import pytest
from selenium.webdriver.common.by import By


class TestClass():

    def test_verify_header_image(self, my_fixture):
        driver = my_fixture
        url = 'https://alchemy.hguy.co/jobs'
        driver.get(url)
        expected_image = 'https://alchemy.hguy.co/jobs/wp-content/uploads/2019/09/career-corporate-job-776615-e1569782235456-1024x271.jpg'
        header_image = driver.find_element(By.XPATH, "//img[@class='attachment-large size-large wp-post-image']")
        actual_image = header_image.get_attribute("src")
        print("Image Source is : " + actual_image)
        assert actual_image == expected_image

File: Selenium/Project/Activity3.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.junit.jupiter.api.Assertions;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

import java.time.Duration;

public class Activity3 {
    WebDriver driver;
    WebDriverWait wait;

    @BeforeClass
    public void setup() {
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs");
        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
    }

    @Test
    public void verifyHeaderImage() {
        WebElement image = driver.findElement(By.xpath("//img[@class='attachment-large size-large wp-post-image']"));
        String imageSrc = image.getAttribute("src");
        System.out.println("Image Source is : " + imageSrc);
    }

    @AfterClass
    public void cleanup() {
        driver.quit();
    }
}
